packages/expo-modules-core/android/src/main/cpp/JSIInteropModuleRegistry.h

```diff
@@ -78,6 +78,8 @@ class JSIInteropModuleRegistry : public jni::HybridClass<JSIInteropModuleRegistr
    */
   jni::local_ref<JavaScriptObject::javaobject> createObject();
 
+  void registerSharedObject(jni::local_ref<jobject> native, jni::local_ref<JavaScriptObject::javaobject> js);
```

Comment:
Could you add a doc comment for this function? It is kinda clear what it does, but would be nice to have it since the other functions in this class are documented.